packages/mobile/src/paymentRequest/types.ts

```diff
@@ -16,7 +16,7 @@ export interface PaymentRequest {
   requesterAddress: string
   requesteeAddress: string
   requesterE164Number?: string
-  currency: SHORT_CURRENCIES
+  currency: Currency
```

Comment:
Don't think we need a migration for the same reason of the others, rehydrate action is not being handled

Reply:
I did add a mapping from old currency to new for backwards compatibility. I'll test this more in depth in the future payment requests specific PR

Reply:
Same comment about `REHYDRATE` action and needing a migration.

`mapOldCurrencyToNew` looks good for backward compat for payment requests from old users to new users.
And looks like we're good for the new users to old users case too (at least for cUSD), since the default `resolveCurrency` defaults to `dollar` when it doesn't recognize the value. 
https://github.com/celo-org/celo-monorepo/blob/c90edf1c938363e24746c90153d36263bb668aef/packages/sdk/base/src/currencies.ts#L42

Plus it seems the UI around payment requests assumes it's all cUSD and doesn't even read the `currency` prop 😅

Reply:
I deleted this prop in a future PR because like you said we don't really use it (we just assume cUSD and that doesn't change with cEUR support since we always show it in local currency) so didn't add a migration for this one.